WeScan/Review/ReviewViewController.swift

```diff
@@ -55,24 +57,29 @@ final class ReviewViewController: UIViewController {
         super.init(nibName: nil, bundle: nil)
     }
     
-    required init?(coder aDecoder: NSCoder) {
+    required public init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    override func viewDidLoad() {
+    override open func viewDidLoad() {
         super.viewDidLoad()
 
         enhancedImageIsAvailable = results.enhancedImage != nil
         
         setupViews()
         setupToolbar()
         setupConstraints()
+    
+        if enhancedImageIsAvailable {
+            isCurrentlyDisplayingEnhancedImage.toggle() //to set right value when the below toggleEnhancedImage is trigger
```

Comment:
Why are we toggling this variable when it previously already had an intended default value? This could change the expected behavior - if we really want to do it we need to mark this as a breaking change.